modules/Microsoft.Compute/images/.test/common/dependencies.bicep

```diff
@@ -30,7 +30,7 @@ resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-
   location: location
 }
 
-resource storageAccount 'Microsoft.Storage/storageAccounts@2021-09-01' = {
+resource storageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' = {
```

Comment:
Same comment of the other PR https://github.com/Azure/ResourceModules/pull/2831#discussion_r1113203070

Reply:
Also same answer :). As agreed in the sync call, resolving the conversation.